src/cpp/src/llm/pipeline.cpp

```diff
@@ -15,6 +15,30 @@
 #include "speculative_decoding/speculative_decoding_impl.hpp"
 #include "utils.hpp"
 
+// Patch properties with safe default values for NPU inference
+// This ensures stable behavior by enabling static pipeline execution
+// and reducing prompt/response sizes to prevent memory issues.
+void patch_npu_properties(ov::AnyMap& properties) {
+    // Enable static pipeline (required for NPU)
+    if (!properties.count("STATIC_PIPELINE"))
+        properties["STATIC_PIPELINE"] = true;
```

Comment:
What are the instability issues with `StatefulPipeline`?

Reply:
Exactly the unified and optimized configuration is chosen by default, so `STATIC_PIPELINE` is not used on purpose.
Please attach numbers that shows the difference between `STATIC_PIPELINE` and not if you found the difference.